content/fr/docs/reference/glossary/cla.md

```diff
@@ -0,0 +1,17 @@
+---
+title: CLA (Contributor License Agreement)
+id: cla
+date: 2018-04-12
+full_link: https://github.com/kubernetes/community/blob/master/CLA.md
+short_description: >
+  Conditions en vertu desquelles un contributeur accorde une licence à un projet open source pour ses contributions.
+
+aka:
+tags:
+- community
+---
+ Conditions en vertu desquelles un contributeur {{< glossary_tooltip text="contributor" term_id="contributor" >}} accorde une licence à un projet open source pour ses contributions.
```

Comment:
```suggestion
 Conditions en vertu desquelles un {{< glossary_tooltip text="contributeur" term_id="contributor" >}} accorde une licence à un projet open source pour ses contributions.
```
?

Reply:
My bad. Now I know. 
I've just translated the glossary_tooltip text section.

Reply:
I've just seen that you've already fixed up this PR. Should I close mine ?

Reply:
This PR is still needed; I only made a _suggested_ change.